website/docusaurus.config.js

```diff
@@ -532,7 +533,10 @@ module.exports = {
                         to: '/',
                     },
                     {
-                        from: '/topics/feature-flags/tutorials',
+                        from: [
+                            '/topics/feature-flags/tutorials',
+                            '/tutorials',
```

Comment:
Are you sure we want this redirect? Will we ever have tutorials that aren't related to feature flags? (Or at least that shouldn't be under that url?)

Reply:
We will probably have others over time, but for now this is the right place. We are planning on building enough of these to merit it's own category. We may add other categories in the future as we test and iterate, but I think this is the best choice now.